syntax = "proto3";

package NGenerativeBoltalka.Proto;

message TBertOutput {
	repeated float Scores = 1 [deprecated = true];
	float RelevScore = 2;
	float InterestScore = 3;
	float NotMaleScore = 4;
	float NotRudeScore = 5;
	float RespectScore = 6;
	float InformativenessScore = 7;
}

message TBertFactorResponse {
	repeated float Scores = 1 [deprecated = true];
    repeated TBertOutput CandidatesScores = 2;
}
